app/models/course.rb

```diff
@@ -106,7 +106,7 @@ def set_subject_position(course_subject)
   has_many :site_statuses
   accepts_nested_attributes_for :site_statuses
   has_many :sites,
-           -> { distinct.merge(SiteStatus.where(status: %i[new_status running])) },
+           -> { kept.distinct.merge(SiteStatus.where(status: %i[new_status running])) },
```

Comment:
Can we have a token test?

Reply:
i don't really see it adding much value? Behaviour has already been tested in other models. I'd just be re-testing implementation logic of the gem at this point